tests/epyccel/test_epyccel_sets.py

```diff
@@ -178,14 +178,14 @@ def Discard_wrong_arg():
     python_result = Discard_wrong_arg()
     assert python_result == pyccel_result
 
-def test_update_basic(python_only_language):
+def test_update_basic(language):
```

Comment:
I think we should support `update` with more than one argument. Is this a choice that has been made?

Reply:
Ah good point, I thought there was something I was supposed to do before marking this as ready. I just couldn't remember what!

Reply:
Fixed : c98b12d

Test added : 5238c30